src/openrct2-ui/windows/Footpath.cpp

```diff
@@ -1139,7 +1139,7 @@ static rct_tile_element * footpath_get_tile_element_to_remove()
             }
         }
     }
-    while (!tile_element_is_last_for_tile(tileElement++));
+    while (!(tileElement++)->IsLastForTile());
```

Comment:
Since we use ++17 now, it may be nicer to rewrite these like:
`while (tileElement++; !tileElement->IsLastForTile())`

Reply:
That would change the meaning - it currently first checks if it's last-for-tile, then increments it.

Reply:
No point doing that if we will eventually move to iterators. See OpenLoco.

Reply:
Right, so could you check if this PR is okay to merge?

Reply:
It looks fine.